src/bundles/EdxAiChatDrawer/EdxAiChatDrawer.mdx

```diff
@@ -0,0 +1,36 @@
+import { Meta, Title, Primary, Controls, Stories } from "@storybook/blocks"
+
+import * as EdxAiChatDrawer from "./EdxAiChatDrawer.stories"
+import { gitLink } from "../../story-utils"
+
+<Meta of={EdxAiChatDrawer} />
+
+<Title />
+
+The EdxAiChatDrawer is an AiChat Drawer intended for use in the EdX `learning` MFE and controlled via problems, videos, and other XBlock content. Because the Xblock content is rendered inside an iframe, the drawer is controlled via `window.parent.postMessage` events.
```

Comment:
I would suggest we remove the `Edx` prefix from the name or replace it with any other prefix like MIT OR OpenedX.